webcompat/static/js/lib/bugform.js

```diff
@@ -221,49 +221,61 @@ function BugForm() {
 
   this.showSimilarBugs = function() {
     if (!$("body").data("username")) {
-      // Not login
+      // Not logged in
       return;
-    }
+    } else {
+      if (this.suggestDelay) {
+        clearTimeout(this.suggestDelay);
+      }
 
-    if (this.suggestDelay) {
-      clearTimeout(this.suggestDelay);
-    }
-    var bugform = this;
-    this.suggestDelay = setTimeout(function() {
-      try {
-        var url = new URL(bugform.urlField.val());
-        if (url.hostname.length > 7) {
-          var query = `https://api.github.com/search/issues?page=1&per_page=10&stage=all&sort=created&q=${url.hostname}+in:title+repo%3Awebcompat%2Fweb-bugs&order=desc`;
-
-          fetch(query)
-            .then(function(response) {
-              var contentType = response.headers.get("content-type");
-              if (contentType && contentType.includes("application/json")) {
-                return response.json();
-              }
-            })
-            .then(function(issues) {
-              if (issues.items) {
-                $("#bugListWrapper").show();
-                $("#bugList").empty();
-                for (var item in issues.items) {
-                  var issue = issues.items[item];
-                  $("#bugList").append(
-                    `<li><a href='https://webcompat.com/issues/${issue.number}' target='_blank'>#${issue.number} - ${issue.title}</a></li>`
-                  );
+      var bugform = this;
+
+      this.suggestDelay = setTimeout(function() {
+        try {
+          var url = new URL(bugform.urlField.val());
+
+          if (url.hostname.length > 3) {
+            var query =
+              "https://api.github.com/search/issues?page=1&per_page=10&stage=all&sort=created&q=" +
+              url.hostname +
+              "in:title+repo%3Awebcompat%2Fweb-bugs&order=desc";
+
+            fetch(query)
+              .then(function(response) {
+                var contentType = response.headers.get("content-type");
+                if (contentType && contentType.includes("application/json")) {
+                  return response.json();
                 }
-              } else {
-                bugform.hideSimilarBugs();
-              }
-            });
-        } else {
+              })
+              .then(function(issues) {
+                if (issues.items) {
+                  $("#bugListWrapper").show();
+                  $("#bugList").empty();
+                  for (var item in issues.items) {
+                    var issue = issues.items[item];
+                    $("#bugList").append(
+                      "<li><a href='https://webcompat.com/issues/" +
```

Comment:
I already left some feedback about this in the last PR, I think using `_.template` from lodash would be better here. No hard objections in a proof of concept branch, but it should be fixed before it gets merged into master.

Reply:
I agree. Will add this to the follow up issue.